libwallet/assets_config.go

```diff
@@ -139,14 +140,25 @@ func (mgr *AssetsManager) GetCurrencyConversionExchange() string {
 	var key string
 	mgr.db.ReadWalletConfigValue(sharedW.CurrencyConversionConfigKey, &key)
 	if key == "" {
-		return "none" // default exchange value
+		return values.DefaultExchangeValue // default exchange value
 	}
 	return key
 }
 
 // SetCurrencyConversionExchange sets the currency conversion exchange.
-func (mgr *AssetsManager) SetCurrencyConversionExchange(data string) {
```

Comment:
what do you think of setting the asset rate as a global variable e.g  `AssetRate             map[libutils.AssetType]float64`. and every page that needs to display the exchange rate gets the value from that map. anywhere the rate gets updated, it is update globally too.

Reply:
Sounds good. I was thinking about an easy to follow framework for ws updates and reading rates info. A global variable will still require mtx locks in multiple places and ws updates will still be needed.

Reply:
Alright then, let's leave as is